### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by andrmaz** [andrmaz/dash-journal#22]
Fix any of the following:\n  Element has insufficient color contrast of 2.95 (foreground color: #c88485, background color: #ffffff, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1
